src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+    >
```

Comment:
The `form` tag is self-closing on line 20, which is incorrect in HTML. The form tag should be opened with `<form>` and closed with `</form>`. This issue is repeated for the second form on line 34.